pkg/sign/certificate.go

```diff
@@ -29,6 +29,10 @@ import (
 	"time"
 )
 
+type Certificate interface {
+	GetCertificate(context.Context, Keypair, string) ([]byte, error)
```

Comment:
We might want to make `string` an `CertificateRequestOptions` struct so that it can be extended in the future. A `KeyPair` will always be needed, but an identity token may not.